lib/output/cli-output.js

```diff
@@ -0,0 +1,256 @@
+'use strict';
+
+const chalk = require('chalk');
+const Table = require('cli-table');
+const {
+  LOG_TITLE,
+  LOG_SEMVER_VIOLATION,
+  LOG_LTS_VIOLATION,
+  LOG_SEMVER_TITLE,
+  LOG_POLICY_TITLE,
+  DEFAULT_SUPPORT_MESSAGE,
+  MILLSINQUARTER
+} = require('./messages');
+
+// CONSTANTS
+const SEMVER_LIST = ['major', 'minor', 'patch'];
+
+function isSemVersion(type) {
+  return SEMVER_LIST.includes(type);
+}
+
+function getTypeForModule(name) {
+  if (name === 'node') {
+    return 'node';
+  } else if (name === 'ember-cli') {
+    return 'ember';
+  }
+}
+
+function isSemVerPolicyViolated(violations) {
+  return Object.keys(violations).some(type => {
+    return violations[type] > 0;
+  });
+}
+
+function getViolations({ violations, violationInfo, totalPackages }) {
+  let violationMessages = {};
+  let isSemVerViolated = isSemVerPolicyViolated(violations);
+  let totalSemVerViolation = 0;
+  for (let type in violations) {
+    let isSemVer = isSemVersion(type);
+    if (isSemVerViolated && isSemVer && violations[type] > 0) {
+      totalSemVerViolation += violations[type];
+      violationMessages[type] = LOG_SEMVER_VIOLATION(type, violations[type], violationInfo[type]);
+    } else if (!isSemVer) {
+      violationMessages[type] = LOG_LTS_VIOLATION(type, violations[type], violationInfo[type]);
+    }
+  }
+
+  // add headline for semVer voilation
+  violationMessages['semVer'] = LOG_SEMVER_TITLE(
+    isSemVerViolated,
+    totalSemVerViolation,
+    totalPackages,
+  );
+
+  return violationMessages;
+}
+
+function getBodyContent(results) {
+  const table = new Table({
+    chars: {
+      top: '',
+      'top-mid': '',
+      'top-left': '',
+      'top-right': '',
+      bottom: '',
+      'bottom-mid': '',
+      'bottom-left': '',
+      'bottom-right': '',
+      left: '',
+      'left-mid': '',
+      mid: '',
+      'mid-mid': '',
+      right: '',
+      'right-mid': '',
+      middle: '',
+    },
+    head: [
+      chalk` {underline {bold {white Name}}}`,
+      chalk`{underline {bold {white Resolved}}}`,
+      chalk`{underline {bold {white Latest}}}`,
+      chalk`{underline {bold {white Behind}}}`,
+    ],
+  });
+  results.forEach(({ name, resolvedVersion, latestVersion, duration, isSupported }) => {
+    if (!isSupported) {
+      table.push([
+        chalk` {red ${name}}`,
+        chalk`{red ${resolvedVersion}}`,
+        chalk`{red ${latestVersion}}`,
+        chalk`{red ${Math.ceil(duration / MILLSINQUARTER)} qtr(s)}`,
+      ]);
+    } else {
+      table.push([
+        chalk` {dim ${name}}`,
+        chalk`{dim ${resolvedVersion}}`,
+        chalk`{dim ${latestVersion}}`,
+      ]);
+    }
+  });
+  return table.toString();
+}
+
+/* Keep the order of voilations summary output as
```

Comment:
```suggestion
/* Keep the order of violations summary output as
```

Reply:
This change is not present in the latest